stdlib/private/StdlibUnittest/StdlibUnittest.swift.gyb

```diff
@@ -578,7 +578,8 @@ extension ProcessTerminationStatus {
       return CInt(signal) == SIGILL || CInt(signal) == SIGTRAP
     default:
       // This default case is needed for standard library builds where
-      // resilience is enabled
+      // resilience is enabled.
+      // FIXME: Need a way to suppress when not.
       return false
```

Comment:
@slavapestov in this case I guess it's in a different module – any options for suppressing it? https://github.com/apple/swift/blob/master/stdlib/private/StdlibUnittest/StdlibUnittest.swift.gyb#L579